src/bindings/python/src/pyopenvino/graph/node.cpp

```diff
@@ -44,35 +45,118 @@ void regclass_graph_Node(py::module m) {
     node.doc() = "openvino.Node wraps ov::Node";
     node.def(
         "__add__",
-        [](const std::shared_ptr<ov::Node>& a, const std::shared_ptr<ov::Node> b) {
-            return std::make_shared<ov::op::v1::Add>(a, b);
+        [](const std::shared_ptr<ov::Node>& left, Common::NodeInput& right) {
+            return std::shared_ptr<ov::Node>(
+                std::make_shared<ov::op::v1::Add>(left, Common::node_from_input_value(right)));
+        },
+        py::arg("right"),
+        R""(
+            Return node which applies f(A,B) = A+B to the input nodes element-wise.
+
+            :param right: The right operand.
+            :type right: Union[openvino.Node, int, float, ndarray]
```

Comment:
```suggestion
            :type right: Union[openvino.Node, int, float, numpy.ndarray]
```
and same in other docstrings